### ISSUE TITLE
Could you clarify the dependencies version?

### ISSUE THREAD

**Issue by Chriszhangmw** [Isvarya12/alzheimers_disease_classification_3ddensenet_selfattention#1]
Hi, thanks for your amazing work,
appreciate if you can help to provide the kinds of packages version.

**Comment by Isvarya12** [comment]
Hi,
The required packages are
scikit-learn 1.2.2
Tensorflow 2.15.0
keras 2.15.0
Hope this helps you! Happy coding !

**Comment by Chriszhangmw** [comment]
Thanks,  after comparing it with this repo: "https://www.kaggle.com/code/lukechugh/99-accuracy-grad-cams"
Your input is 2D images, right? If so, where did you convert the 3D to 2D? From what I gather, your work utilizes the 3D densenet(your title mentions this), where 3D refers to the input being *.nii files. I'm new to this field, so thank you for your explanation if possible.

**Comment by Isvarya12** [comment]
Yes you are right, I thought it would be easy to work with 2D images. You
can refer the attached file to convert *nii to .png .
Hope this helps.

**Comment by Isvarya12** [comment]
Awesome, thanks for the update!

**Comment by Isvarya12** [comment]
If you find this repo to be useful, please give it a star :)
